src/openrct2/park/Legacy.cpp

```diff
@@ -2275,7 +2276,7 @@ void UpdateFootpathsFromMapping(
     pathToRailingsMap[entryIndex] = railingIndex;
 }
 
-const std::vector<std::string_view> peepAnimObjects = {
+static constexpr auto kPeepAnimObjects = std::to_array<std::string_view>({
```

Comment:
Nice catch. In the first version, this used to be populated dynamically based on theme selection, but I clearly forgot to change the container type when that was abolished...